application_manager/utils/mesos.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2017 UFCG-LSD.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+import json
+
+def extract_vms_ids(output):
+    lines = output.split('\n')
+    ids = []
+    for i in range(1, len(lines)-1):
+        ids.append(lines[i].split()[0])
+
+    return ids
+
+
+def get_executors_ip(conn, get_frameworks_url, app_id):
+    frameworks_call = ('curl http://' + get_frameworks_url + '/frameworks')
+
+    time.sleep(5)
+    stdin, stdout, sterr = conn.exec_command(frameworks_call)
+
+    try:
+        output = stdout.read()
+        mesos_resp = json.loads(output)
+    except Exception as e:
```

Comment:
application_manager/utils/mesos.py:37:25: F841 local variable 'e' is assigned to but never used
    except Exception as e:
                        ^

Talvez atualizaro except para mostrar a mensagem da excecao mesmo

Reply:
Print added.